spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from spidermon.contrib.actions.templates import ActionWithTemplates
+from spidermon.exceptions import NotConfigured
+
+
+class SendMessage(ActionWithTemplates):
```

Comment:
This action doesn't render any template, so it doesn't need to inherit from it. Is there any other reason to use this inheritance?

Reply:
The actual reason is to get context data which is being sent to these templates.

Reply:
The context used in `get_message` is provided by any `Action`, not only `ActionWithTemplates` (`self.result` and `self.data` contains everything needed). For me, the title rendering method is unnecessary, as it is a simple string. IMHO, inheriting only from `Action` will make the action simpler, as other user will not struggle trying to find ways to use templates.